tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -731,6 +754,13 @@ private void SetupGitlabEnvironment()
 
             // Clean pipeline url
             PipelineUrl = PipelineUrl?.Replace("/-/pipelines/", "/pipelines/");
+
+            // Node
+            NodeName = EnvironmentHelpers.GetEnvironmentVariable(Constants.GitlabRunnerId);
+            if (EnvironmentHelpers.GetEnvironmentVariable(Constants.GitlabRunnerTags) is { } runnerTags)
+            {
+                NodeLabels = Datadog.Trace.Vendors.Newtonsoft.Json.JsonConvert.DeserializeObject<string[]>(runnerTags);
```

Comment:
FYI this could throw - probably needs a try-catch?

Reply:
done in [df3a6a6](https://github.com/DataDog/dd-trace-dotnet/pull/4049/commits/df3a6a68b29d2d548bd006e1204e9c793b87c18e)